tests/plugins/test_plugins_manager.py

```diff
@@ -64,3 +65,28 @@ def test_app_blueprints(self):
         # Blueprint should be present in the app
         self.assertTrue('test_plugin' in self.app.blueprints)
         self.assertEqual(self.app.blueprints['test_plugin'].name, bp.name)
+
+
+@mock.patch('airflow.plugins_manager.pkg_resources.iter_entry_points')
```

Comment:
```suggestion
@mock.patch('airflow.plugins_manager.pkg_resources.iter_entry_points')
@mock.patch('airflow.plugins_manager. import_errors', {})
```
It prevents side effects.

Reply:
Fixed